docs/source/gateway/index.rst

```diff
@@ -618,31 +618,38 @@ Querying the AI Gateway
 
 Once the MLflow AI Gateway server has been configured and started, it is ready to receive traffic from users.
 
-Query Parameters
-----------------
+.. _standard_query_parameters:
+
+Standard Query Parameters
+-------------------------
 
 The query parameters that are supported by various providers for different route types are also available to be used with the MLflow AI Gateway.
 Each of these query parameters are optional elements that can be included along with using the ``query`` APIs as key value pairs within the ``data`` argument.
 The AI Gateway will perform validation of these commonly modified parameters to ensure that provider-specific restrictions and scaling factors are unified with a consistent range of allowable values.
 If a given provider does not provide support for a parameter, a clear message will be returned when queried that explains the restrictions for the given provider and route type.
 
-- **temperature** (Supported by OpenAI, Anthropic, Cohere): This parameter controls the randomness of predictions by scaling the logits before applying softmax. A value closer to 0.0 makes the output more deterministic, while a value closer to 1.0 makes it more diverse. Default is 0.0.
+- ``temperature``: This parameter controls the randomness of predictions by scaling the logits before applying softmax. A value closer to 0.0 makes the output more deterministic, while a value closer to 1.0 makes it more diverse. Default is 0.0.
+
+- ``max_tokens``: This parameter limits the length of the generated output by specifying a maximum token count. The range is from 1 to infinity, and by default, there is no limit (infinity). Some providers have a maximum value associated with this parameter that the AI Gateway will enforce to prevent a provider-generated exception.
 
-- **max_tokens** (Supported by OpenAI, Anthropic, Cohere): This parameter limits the length of the generated output by specifying a maximum token count. The range is from 1 to infinity, and by default, there is no limit (infinity). Some providers have a maximum value associated with this parameter that the AI Gateway will enforce to prevent a provider-generated exception.
```

Comment:
Changed from bold to code format.

Also removed "Supported by". This is confusing since these are standard parameters...they should be supported by every provider.

It's true that Anthropic only supports candidate count 1, but that argument can still be specified.